src/main/java/io/spaship/operator/service/k8s/CommandExecutionService.java

```diff
@@ -83,8 +83,9 @@ private String[] symbolicLinkCommand(String source, String target) {
         source = (BASE_HTTP_DIR.concat("/").concat(source));
         target = (BASE_HTTP_DIR.concat("/").concat(target));
         LOG.debug("creating a symlink of source {} to {}",source,target);
-        LOG.debug("command to be executed [ln] [-s] [{}] [{}]",source,target);
-        return new String[]{"ln", "-s", source, target};
+        String command = "rm -f " + target + " && ln -s " + source + " " + target;
```

Comment:
@arkaprovob Code looks good to me, but I've once been concerned about this line. If a user puts a target path that doesn't exist as a symlink but exists as a file, It will end up deleting that file.

In the future improvement, can we check if that target exists as a symlink, then only we'll delete it or else we'll throw an error !

also evey-time symlink won't be a file, it can be a folder also, so in that case this code might throw an error !

```
rm -f innter
rm: innter: is a directory
```